Index: frameworkintegration-4.95.0/autotests/CMakeLists.txt
===================================================================
--- frameworkintegration-4.95.0.orig/autotests/CMakeLists.txt	2014-01-03 12:00:34.000000000 +0000
+++ frameworkintegration-4.95.0/autotests/CMakeLists.txt	2014-02-04 17:00:22.950647797 +0000
@@ -18,14 +18,6 @@
 endmacro()
 
 frameworkintegration_tests(
-  kdeplatformtheme_unittest
-  ../src/platformtheme/kdeplatformtheme.cpp
-  ../src/platformtheme/kfontsettingsdata.cpp
-  ../src/platformtheme/khintssettings.cpp
-  ../src/platformtheme/kdeplatformfiledialoghelper.cpp
-)
-
-frameworkintegration_tests(
   kfontsettingsdata_unittest
   ../src/platformtheme/kfontsettingsdata.cpp
 )
